### PR TITLE
docs: add marketplace commands, capitalize Mia-Platform related terms

### PR DESCRIPTION
**Pull request by malta895**
## What this PR is for?
- Fixes to words that should be capitalized
- Add `miactl marketplace` subcommands documentation

### CODE REVIEW

File: docs/40_examples.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Create a Context for a company on the cloud instance:
 miactl context set paas-company --company-id <your-company-id>
 ```
 
-Create a Context for specific project in a company on the cloud instance:
+Create a Context for specific project bin a ompany on the cloud instance:
```
[the remark this answers is not in the document]
Comment (malta895) [reply docs/40_examples.md:14]:
```suggestion
Create a Context for specific project in a Company on the cloud instance:
```

File: docs/30_commands.md [hunk 2]
```diff
@@ -188,3 +196,93 @@ Available flags for the command:
...
+
+# Apply all the valid configuration files in the directory myFantasticGoTemplates to the Marketplace
+miactl marketplace apply -d myFantasticGoTemplates
+```
+
+#### Options
+
+```
+  -d, --directory string   a path to a directory containing Marketplace resource files
+  -f, --file stringArray   a path to a JSON or YAML file containing a Marketplace resource
```
Comment (pmarchini) [inline docs/30_commands.md:286]: @malta895, are these options still correct, or is an update needed?

Comment (malta895) [reply docs/30_commands.md:286]: Yes an update is needed. I'll integrate it with the image upload functionality, as soon as it is finished

File: docs/30_commands.md [hunk 3]
```diff
@@ -226,5 +234,109 @@ Available flags for the command:
...
+	"localPath": "./someImage.png"
+}
+The localPath can be absolute or relative to the file location.
+The image will be uploaded along with the Marketplace item.
+Before being applied, the "image" key will be replaced with the "imageUrl" referring to the uploaded image.
+You can retrieve the updated item with the "get" command.
+
+You can also specify the "supportedByImage" in a similar fashion.
+
+Be aware that the presence of both "image" and "imageUrl" and of both "supportedByImage" and "supportedByImageUrl" is illegal.
```
Comment (malta895) [inline docs/30_commands.md:316]:
```suggestion
Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is illegal.
```

File: docs/30_commands.md [hunk 4]
```diff
@@ -226,5 +234,109 @@ Available flags for the command:
...
+The file can contain an image object with the following format:
+"image": {
+	"localPath": "./someImage.png"
+}
+The localPath can be absolute or relative to the file location.
+The image will be uploaded along with the Marketplace item.
+Before being applied, the "image" key will be replaced with the "imageUrl" referring to the uploaded image.
+You can retrieve the updated item with the "get" command.
+
+You can also specify the "supportedByImage" in a similar fashion.
```
Comment (marcofilippi) [inline docs/30_commands.md:314]:
```suggestion
You can also specify the "supportedByImage" in a similar way.
```

File: docs/30_commands.md [hunk 5]
```diff
@@ -226,5 +234,109 @@ Available flags for the command:
...
+
+Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is illegal.
```
Comment (marcofilippi) [inline docs/30_commands.md:316]: may opt for "is not allowed" instead

Comment (malta895) [reply docs/30_commands.md:316]: I wanted to emphasize the fact that doing it raises an error. Maybe we can do this?
```suggestion
Be aware that the presence of both "image" and "imageUrl" and/or of both "supportedByImage" and "supportedByImageUrl" is ambiguous and raises an error .
```

**Review by JGiola — COMMENTED** [review]
Can you rebase from the latest commits in main to run a pass with the new docs recently merged?

**Comment by malta895** [comment]
> Can you rebase from the latest commits in main to run a pass with the new docs recently merged?

I think I already did it yesterday, can you check again?

**Comment by malta895** [comment]
> My bad, I thought that another PR were merged, but in fact is not. If all the comments are closed I will merge it.

OK no problem!  There is one by @marcofilippi, as soon as it is resolved we can merge!

**Review by JGiola — APPROVED** [review]

**Pull request merged**